### ISSUE TITLE
Tracking issue for sidecar -> waypoint interop

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#51445]
Details TBD, mostly opening this as a TODO reference in tests for now

**Comment by keithmattix** [comment]
I think over time we've gotten pretty close to getting all the pieces we need for this. The logic behind ingress-use-waypoint will work for identifying waypoints and we already have hbone sending

**Comment by djahandarie** [comment]
@keithmattix Just curious, has there been any progress on this? If I understand correctly, this would be a blocker for any organization trying to migrate from sidecar to ambient if they are using L7 features of istio (which are normally handled by the destination service, like AuthorizationPolicy), since the waypoint would be bypassed in the case of a hybrid setup, right? I imagine most organizations would not be able to do a hard cut of all services from sidecar to ambient. I'm curious how this is being prioritized within the istio team.

**Comment by Spitfire1900** [comment]
Agreeing with @djahandarie here. It's pretty critical that an organization be able to slowly cut over to Ambient per workload, and for some use cases stick with proxy sidecar for select workloads in a mesh long term.

**Comment by keithmattix** [comment]
@krinkinmu @fjglira and others are looking into it

**Comment by davidspek** [comment]
Is anybody still working on this?

**Comment by krinkinmu** [comment]
Yes, we are looking at the sidecar and ambient interop in context of the migration from sidecar to ambient, plus there are few specific issues related to ambient and sidecar co-existing together that we are working on. @davidspek do you have a specific issue you're having?
